packages/codemirror/src/cypher.js

```diff
@@ -150,5 +150,8 @@ export const cypher = {
     if (context.type === "keywords") return null; // return CodeMirror.commands.newlineAndIndent;
     if (context.align) return context.col + (closing ? 0 : 1);
     return context.indent + (closing ? 0 : cx.unit);
+  },
+  languageData: {
```

Comment:
What a simple & elegant fix to the problem, great work! I tested with & without this change and confirmed that it fixes commenting to work as expected.

How did you know to add the languageData like that? ;-)